### PR TITLE
Move History to Activity.tsx

### PR DESCRIPTION
**Pull request by oneleo**
## InfoNavBar Preview

- Localnet

image

- Sepolia

### CODE REVIEW

File: app/page/info/index.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    <NavbarLayout>
+      {account.address && (
+        <div className="flex justify-center items-center h-auto p-3 border-0 rounded-lg text-base">
+          <button
+            onClick={handleClick}
+            data-url={`${explorerUrl}account/${
+              account.address
+            }?network=${getChainName(account.chainId)}`}>
+            {`${account.address}`}
+          </button>
```
Comment (pilagod) [inline app/page/info/index.tsx:84]: Same for the activity tab. And the `handleClick` function could be removed altogether.

```suggestion
          <a
            href={`${explorerUrl}account/${
              account.address
            }?network=${getChainName(account.chainId)}`}
            target="_blank"
            >
            {account.address}
          </button>
```

File: app/page/info/index.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        </div>
+      )}
+      <div className="flex justify-center items-center h-auto p-3 border-0 rounded-lg text-base">
+        Balance: {balanceLoading ? "(Loading...)" : ethers.formatEther(balance)}
+      </div>
+      <SwitchToSendPage />
+      <InfoNavBar />
```
Comment (pilagod) [inline app/page/info/index.tsx:91]: Could be better to inline them. If you want, you can add `{/* Comment */}` before each block to annotate.

File: app/page/info/index.tsx (hunk 1 above)
Comment (oneleo) [reply app/page/info/index.tsx:84]: It doesn't seem to work...
Clicking the link to the URL directly in the `Popup Script` doesn't display the URL link. Currently,
the only known way is to create a new window, which will display the URL link in the adjacent `Chrome` window.

Or should I add the new window creation function to the `<button>` `onClick` directly?
```typescript
          <button
            onClick={() => {
              browser.tabs.create({
                url: `${explorerUrl}account/${
                  account.address
                }?network=${getChainName(account.chainId)}`
              })
            }}>
            {`${account.address}`}
          </button>
```

File: app/page/info/index.tsx (hunk 2 above)
Comment (oneleo) [reply app/page/info/index.tsx:91]: Does it mean I can separate them like below?
```typescript
      {/* SwitchToSendPage */}
      <div>...</div>
      {/* InfoNavBar */}
      <div>...</div>
```

File: app/page/info/index.tsx (hunk 1 above)
Comment (pilagod) [reply app/page/info/index.tsx:84]: I had tried in my browser and it works. Maybe we can discuss it this afternoon.

**Review by pilagod — APPROVED** [review]

**Pull request merged**